python/ray/train/tests/test_gpu.py

```diff
@@ -111,14 +111,13 @@ def train_fn():
 
     count = Counter(devices)
     if num_gpus_per_worker == 0.5:
-        for i in range(4):
+        for i in range(2):
             assert count[i] == 4
```

Comment:
can you add a comment in the code about the expected setup (workers <> gpus topology)?

Reply:
done